mcs/class/Mono.Posix/Mono.Unix.Native/Syscall.cs

```diff
@@ -4545,9 +5166,36 @@ public static long pwritev (int fd, Iovec[] iov, long offset)
 		}
 		#endregion
 
+		#region <arpa/inet.h> Declarations
+		//
+		// <arpa/inet.h>
+		//
+
+		// htonl(3)
+		//    uint32_t htonl(uint32_t hostlong);
+		[DllImport (LIBC)]
+		public static extern uint htonl(uint hostlong);
```

Comment:
I thought mentioned elsewhere/previously, but I'm not sure we need to bind these as identical functionality is available from [IPAddress.NetworkToHostOrder()](https://msdn.microsoft.com/en-us/library/system.net.ipaddress.networktohostorder(v=vs.110).aspx), except the methods here operate on unsigned t types.